docs/source/conf.py

```diff
@@ -340,3 +341,41 @@
 
 spelling_lang = 'en_US'
 spelling_word_list_filename = 'spelling_wordlist.txt'
+
+
+def _import_object_from_name(module_name, fullname):
+    obj = sys.modules.get(module_name)
+    if obj is None:
+        return None
+    for comp in fullname.split('.'):
+        obj = getattr(obj, comp)
+    return obj
+
+
+def linkcode_resolve(domain, info):
+    if domain != 'py' or not info['module']:
+        return None
+
+    tag = 'v{}'.format(__version__)
```

Comment:
I thought it's ok because we only release the documentation at each of tagged versions.
(Maybe I don't get your point?)

Btw, `READTHEDOCS_VERSION` cannot be used in local build. Is it OK?